### DISCUSSION TITLE
Error in Top scores of the Target 100

### DISCUSSION THREAD

**Discussion by SelenIT** [cssbattle/forum#202] [Bugs reports]
When I try to load the Top scores for the [Open Target #100](https://cssbattle.dev/play/100), no data gets shown, the loading indicator keeps spinning infinitely instead and the following error appears in the console:
```
index.cjs.js:160 
        
       Uncaught (in promise) FirebaseError: Missing or insufficient permissions.
    at new e (index.cjs.js:160)
    at t._n (index.cjs.js:3482)
    at t.kn (index.cjs.js:3646)
    at e.onMessage (index.cjs.js:7026)
    at index.cjs.js:6979
    at index.cjs.js:7010
    at index.cjs.js:4986
```

Could you please look into this issue?

**Comment by chinchang** [comment]
Fixed now. Sorry about that!

**Reply by SelenIT** [reply]
Thanks a lot!
